addons/project/models/project_task.py

```diff
@@ -190,7 +190,7 @@ def _read_group_personal_stage_type_ids(self, stages, domain, order):
     displayed_image_id = fields.Many2one('ir.attachment', domain="[('res_model', '=', 'project.task'), ('res_id', '=', id), ('mimetype', 'ilike', 'image')]", string='Cover Image')
 
     is_closed = fields.Boolean(compute='_compute_is_closed', string="Closed State", store=True, index=True)
-    parent_id = fields.Many2one('project.task', string='Parent Task', index=True)
+    parent_id = fields.Many2one('project.task', string='Parent Task', index=True, domain="[('id', '!=', id)]")
```

Comment:
```suggestion
    parent_id = fields.Many2one('project.task', string='Parent Task', index=True, domain="['!', ('id', 'child_of', id)]")
```
I discussed this with AJU, and we agree that it would be even better to hide all the descendants :slightly_smiling_face:

Reply:
Hello @MissingNoShiny 
Thank you for looking into it.I have implemented the changes